readme.txt

```diff
@@ -0,0 +1,106 @@
+=== Infinum WordPress Boilerplate ===
+Contributors: mustra, dingo_bastard, ivangrginov
+Tags: Starter Theme, Webpack, Node, NodeJS, npm, npx, Build Process, Autoprefixr, SCSS, SASS, LESS, Babel, Browsersync, Accessibility Ready, Custom Colors, Custom Header, Custom Logo, Custom Menu, Editor Style, Featured Images, Flexible Header, Footer Widgets, One Column, Theme Options, Translation Ready, Two Columns
```

Comment:
Be very careful when adding `Accessibility Ready` tag. As any theme submitted with this tag has to go through a special review on wordpress.org.

Also, there seem to be some nonstandard tags here, which could trigger some reviewers:

[Rule about tags in style.css and description](https://make.wordpress.org/themes/handbook/review/required/#core-functionality-and-features)
[Theme tags](https://make.wordpress.org/themes/handbook/review/required/theme-tags/)

Reply:
👍 Removed most of the tags since we don't support pretty much anything in the backend without coding it yourself :)